site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Segmented Controls
+description: Segmented controls component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - segmented-control
+  - segmented-controls
+  - segmented-controls-after
```

Comment:
Suggest either updating `segmented-controls-after` to `segmented-controls-footer`, or just removing altogether.

Reply:
✅